deploy/README.md

```diff
@@ -361,8 +361,9 @@ VALIDATORS=0x 0x 0x
 HOME_REWARDABLE=false
 # The flag defining whether to use RewardableValidators contract and set a fee manager contract on Foreign network
 FOREIGN_REWARDABLE=false
-# Flag to define if Home network is a POSDAO and rewards are distributed by blockReward contract
-HOME_POSDAO=false
+# Variable to define if Home network is a POSDAO and rewards are distributed by blockReward contract to network validators or transferred directly to bridge validators.
+# Supported values are BRIDGE_VALIDATORS_REWARD and POSDAO_REWARD
+HOME_FEE_MANAGER_TYPE=BRIDGE_VALIDATORS_REWARD
```

Comment:
We need to specify in the examples of `.env` file that this value must be commented.

Reply:
Updated `.env.example`. Please check if it looks good to you https://github.com/poanetwork/poa-bridge-contracts/pull/164/commits/cad1cc5c2b436c6364d77239430e0ec0224cedc5

Reply:
thanks